src/python/grpcio/grpc/_cython/_cygrpc/aio/server.pyx.pxi

```diff
@@ -174,6 +208,15 @@ cdef class _ServicerContext:
     def set_details(self, str details):
         self._rpc_state.status_details = details
 
+    def set_compression(self, object compression):
+        if self._rpc_state.metadata_sent:
+            raise RuntimeError('Compression setting must be specified before sending initial metadata')
+        else:
+            self._rpc_state.compression_algorithm = compression
+
+    def disable_next_message_compression(self):
+        self._rpc_state.disable_next_compression = True
```

Comment:
I'm a bit lost with that method, what does it really do? and moreover, in what circumstances is useful?

Reply:
This function is useful against attacks to the compression traffic: https://github.com/grpc/grpc/blob/master/doc/compression.md#specific-disabling-of-compression

This function marks that the next message should be sent without compression, and it should work for unary response and stream responses.